src/components/ProjectContent.jsx

```diff
@@ -0,0 +1,60 @@
+import projectimg from '../assets/project.jpg';
+
+function ProjectContent() {
+  return (
+    <div className="grid grid-cols-1 sm:grid-cols-1 md:grid-cols-2 gap-8  p-3">
+      <div className="flex flex-col p-3 ">
+        <img
+          src={projectimg}

```

Comment:
Let's make this a prop that gets passed into this component.

Reply:
done